src/components/Detailpage/Actions.tsx

```diff
@@ -0,0 +1,53 @@
+import Button from '@mui/material/Button';
+import { MoreHoriz } from '@mui/icons-material';
+import { Stack } from '@mui/material';
+import { spacings } from '../DesignSystem/DesignSystem';
+import { MouseEvent, useState } from 'react';
+import Menu from '../Menu/Menu';
+import { MenuItems } from '../../types/types';
+import { useNavigate } from 'react-router-dom';
+import { useConfig } from '../Config/Config';
+
+export default function DetailpageActions({ concept }: { concept?: boolean }) {
```

Comment:
Still in the idea of Sven's comment that components in the `components` folder should be generic, I think the props could be something like `canChange` and `canChangeTextButton` (or some better name... :stuck_out_tongue: ) and then L48 could be changed accordingly.

Reply:
I resolved this component in the `ZaaktypenView`.